src/trans_table.rs

```diff
@@ -111,10 +111,13 @@ impl TranslationTable {
 
         let mut result = Vec::with_capacity(dna.len() / 3);
 
-        // this will truncate any trailing non-multiple-of-3 chunk
-        // biopython also truncates, but warns -- generally I don't think we care,
-        // so I just made it silently truncate
-        for &chunk in dna.array_chunks::<3>() {
+        for chunk in dna.chunks(3) {
```

Comment:
Can you use `chunks_exact` https://doc.rust-lang.org/std/primitive.slice.html#method.chunks_exact to omit the check?